internal/apis/core/v1alpha3/condition_types.go

```diff
@@ -72,7 +72,7 @@ type Condition struct {
 	// Last time the condition transitioned from one status to another.
 	// This should be when the underlying condition changed. If that is not known, then using the time when
 	// the API field changed is acceptable.
-	// +required
+	// +optional
```

Comment:
Why do we need this change?

Reply:
New ct version makes the field required otherwise. Required marker now has a higher priority then omitempty

Reply:
I think it's more important that our CRDs don't change vs. our internal types